source/dev_section/programming_tutorials/programming_tutorial2/setup_pycharmproject.rst

```diff
@@ -15,7 +15,7 @@ Setup your programming project
     .. code-block:: batch
 
         cd <workshop folder>
-        git clone https://bitbucket.org/hu-geomatics/enmap-box-workshop2019.git
```

Comment:
Better leave as was before. This tutorial is pretty outdated (there is no enmap-box-workshop2019 repository)